packages/g/glslang/xmake.lua

```diff
@@ -10,10 +9,15 @@ package("glslang")
     add_versions("1.2.189+1", "2fb89a0072ae7316af1c856f22663fde4928128a")
     add_versions("1.3.211+0", "9bb8cfffb0eed010e07132282c41d73064a7a609")
     add_versions("1.3.231+1", "5755de46b07e4374c05fb1081f65f7ae1f8cca81")
+    add_versions("1.3.236+0", "77551c429f86c0e077f26552b7c1c0f12a9f235e")
+    add_versions("1.3.239+0", "ca8d07d0bc1c6390b83915700439fa7719de6a2a")
 
     add_configs("binaryonly", {description = "Only use binary program.", default = false, type = "boolean"})
     add_configs("exceptions", {description = "Build with exception support.", default = false, type = "boolean"})
     add_configs("rtti",       {description = "Build with RTTI support.", default = false, type = "boolean"})
+    if is_plat("wasm") then
+        add_configs("shared", {description = "Build shared library.", default = false, type = "boolean", readonly = true})
+    end
```

Comment:
Is it possible for wasm to use dynamic libraries? If it only supports static, I think we can disable shared for wasm inside xmake.

Reply:
it does but it's a bit complicated: https://emscripten.org/docs/compiling/Dynamic-Linking.html

Reply:
oh, got it, thanks